app/views/layouts/application.html.erb

```diff
@@ -20,7 +20,11 @@
     <% end %>
     <%= navbar_group align: 'right' do %>
       <% if user_signed_in? %>
-        <li><%= link_to 'Logout', destroy_user_session_path, method: :delete %></li>
+        <%= navbar_dropdown @user.name do %>
+          <%= navbar_text @user.email %>
+          <%= navbar_dropdown_divider %>
+          <%= navbar_item (link_to 'Logout', destroy_user_session_path, method: :delete) %>
```

Comment:
This should work, but I just wanted to give you a heads up on a Ruby weirdness:

`navbar_item (link_to 'Logout', destroy_user_session_path, method: :delete)` and `navbar_item(link_to 'Logout', destroy_user_session_path, method: :delete)` are distinct. In the first, the parens are normal grouping parens, and all of the arguments go to `link_to`. In the second, the parens are function-parameter parens, and `navbar_item` gets three arguments.